src/calculateRentalCost.js

```diff
@@ -28,7 +28,16 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const price = days * 40;
+
+  switch (true) {
+    case days >= 7:
+      return price - 50;
+    case days >= 3:
+      return price - 20;
+    default:
+      return price;
+  }
```

Comment:
This is not the best way to `use switch/case` structure.

Rewrite it using `if`

Reply:
Ok) I will fix it. But I don't understand why I can't use switch/case for this task?

Reply:
Switch with `true` works, but it should not be used this way.
It contradicts the logic of `switch/case` purpose where only one case should be true.